tests/snuba/api/endpoints/test_organization_sessions.py

```diff
@@ -0,0 +1,411 @@
+        self.project2 = self.create_project()
+        self.project3 = self.create_project()
+        self.project4 = self.create_project(organization=self.organization2)
+
+        template = {
+            "distinct_id": "00000000-0000-0000-0000-000000000000",
+            "status": "exited",
+            "seq": 0,
+            "release": "foo@1.0.0",
+            "environment": "production",
+            "retention_days": 90,
+            "duration": None,
+            "errors": 0,
+            "started": self.session_started,
+            "received": self.received,
+        }
+
+        def make_session(project, **kwargs):
+            return dict(
+                template,
+                session_id=uuid4().hex,
+                org_id=project.organization_id,
+                project_id=project.id,
+                **kwargs
+            )
+
+        self.store_session(make_session(self.project1))
+        self.store_session(make_session(self.project1, release="foo@1.1.0"))
+        self.store_session(make_session(self.project1, started=self.session_started - 60 * 60))
+        self.store_session(make_session(self.project1, started=self.session_started - 12 * 60 * 60))
+        self.store_session(make_session(self.project2, status="crashed"))
+        self.store_session(make_session(self.project2, environment="development"))
+        self.store_session(make_session(self.project3, errors=1))
+        self.store_session(
+            make_session(
+                self.project3,
+                distinct_id="39887d89-13b2-4c84-8c23-5d13d2102664",
+                started=self.session_started - 60 * 60,
+            )
+        )
+        self.store_session(
+            make_session(
+                self.project3, distinct_id="39887d89-13b2-4c84-8c23-5d13d2102664", errors=1
+            )
+        )
+        self.store_session(make_session(self.project4))
+
+    def do_request(self, query):
+        self.login_as(user=self.user)
+        url = reverse(
+            "sentry-api-0-organization-sessions",
+            kwargs={"organization_slug": self.organization.slug},
+        )
+        return self.client.get(url, query, format="json")
+
+    def test_empty_request(self):
+        response = self.do_request({})
+
+        # TODO: making an empty request throws currently
+        assert response.status_code == 500, response.content
+        assert response.data["detail"] == "Internal Error"
+
+    def test_inaccessible_project(self):
+        response = self.do_request({"project": [self.project4.id]})
+
+        assert response.status_code == 403, response.content
+        assert response.data == {"detail": "You do not have permission to perform this action."}
+
+    def test_unknown_field(self):
+        response = self.do_request({"field": ["summ(sessin)"]})
+
+        # TODO: throws currently
+        assert response.status_code == 500, response.content
+        assert response.data == {"detail": "Internal Error", "errorId": None}
```

Comment:
You might have to only assert the `detail` here, or catch the internal error and return a 400 response.